src/rhc.erl

```diff
@@ -109,14 +111,16 @@ create() ->
 %% @spec create(string(), integer(), string(), Options::list()) -> rhc()
 create(IP, Port, Prefix, Opts0) when is_list(IP), is_integer(Port),
                                      is_list(Prefix), is_list(Opts0) ->
-    Opts = case proplists:lookup(client_id, Opts0) of
-               none -> [{client_id, random_client_id()}|Opts0];
-               Bin when is_binary(Bin) ->
```

Comment:
How could that have worked, since one would always have been a tuple returned....

Reply:
The create option with a pre-defined client_id is a legacy of very, very old riak when the vector clocks client ids would come from the application not the vnode - I think.  It is almost certainly a dead bit of code.  I guess really it should be removed